<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>557</width>
    <height>747</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>511</width>
     <height>721</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>17</number>
    </property>
    <property name="topMargin">
     <number>40</number>
    </property>
    <property name="bottomMargin">
     <number>9</number>
    </property>
    <property name="verticalSpacing">
     <number>6</number>
    </property>
    <item row="7" column="2">
     <widget class="QComboBox" name="item_box"/>
    </item>
    <item row="3" column="2">
     <widget class="QComboBox" name="main_menu"/>
    </item>
    <item row="2" column="0" colspan="3">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Select Main Menu</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="12" column="4">
     <widget class="QPushButton" name="btn_square">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>140</width>
        <height>120</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>140</width>
        <height>120</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:10;
background-color: rgb(0, 0, 255);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
    </item>
    <item row="9" column="0" colspan="5">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Select Button Type</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QComboBox" name="sub_cat_box"/>
    </item>
    <item row="0" column="4">
     <widget class="QPushButton" name="place_btn">
      <property name="minimumSize">
       <size>
        <width>140</width>
        <height>60</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>140</width>
        <height>60</height>
       </size>
      </property>
      <property name="text">
       <string>Place button</string>
      </property>
     </widget>
    </item>
    <item row="14" column="4">
     <widget class="QPushButton" name="cancel_btn">
      <property name="minimumSize">
       <size>
        <width>140</width>
        <height>60</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>140</width>
        <height>60</height>
       </size>
      </property>
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="3">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>CREATE OR MODIFY BUTTONS</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="3">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Select Category </string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="12" column="2">
     <widget class="QPushButton" name="btn_rectangular">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>140</width>
        <height>80</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>140</width>
        <height>80</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:10;
background-color: rgb(0, 0, 255);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
    </item>
    <item row="14" column="3">
     <widget class="QPushButton" name="save_btn">
      <property name="minimumSize">
       <size>
        <width>140</width>
        <height>60</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>140</width>
        <height>60</height>
       </size>
      </property>
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0" colspan="3">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Select Item</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="14" column="2">
     <widget class="QPushButton" name="delete_btn">
      <property name="minimumSize">
       <size>
        <width>140</width>
        <height>60</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>140</width>
        <height>60</height>
       </size>
      </property>
      <property name="text">
       <string>Delete Button</string>
      </property>
     </widget>
    </item>
    <item row="13" column="0" colspan="5">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Minimum</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>70</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="0" colspan="5">
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>60</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="6" column="4">
     <widget class="QComboBox" name="text_color">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="4" column="4">
     <widget class="QComboBox" name="bg_color">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="3" column="3" colspan="2">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Select BackGround</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="5" column="3" colspan="2">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Select Text-Color</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="12" column="3">
     <widget class="QRadioButton" name="check_btn_2">
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="12" column="0" colspan="2">
     <widget class="QRadioButton" name="check_btn_1">
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
